packages/e2e-tests/specs/editor/various/multi-block-selection.test.js

```diff
@@ -1,1048 +0,0 @@
-		const coord2 = await paragraph2.clickablePoint();
-
-		await page.mouse.move( coord1.x, coord1.y );
-		await page.mouse.down();
-		await page.mouse.move( coord2.x, coord2.y, { steps: 10 } );
-		await page.mouse.up();
-
-		await testNativeSelection();
-		expect( await getSelectedFlatIndices() ).toEqual( [ 1, 2 ] );
-	} );
-
-	it( 'should select by dragging into nested block', async () => {
-		await clickBlockAppender();
-		await page.keyboard.type( '1' );
-		await page.keyboard.press( 'Enter' );
-		await page.keyboard.type( '/group' );
-		await page.waitForXPath(
-			'//button[@aria-selected="true"][text()="Group"]'
-		);
-		await page.keyboard.press( 'Enter' );
-
-		// Select the default, selected Group layout from the variation picker.
-		await page.click(
-			'button[aria-label="Group: Gather blocks in a container."]'
-		);
-
-		const groupAppender = await page.waitForSelector(
-			'.block-editor-button-block-appender'
-		);
-		await groupAppender.click();
-
-		const paragraphBlockButton = await page.waitForSelector(
-			'.editor-block-list-item-paragraph'
-		);
-		await paragraphBlockButton.click();
-
-		await page.keyboard.type( '2' );
-
-		const [ paragraph1, paragraph2 ] = await page.$$(
-			'[data-type="core/paragraph"]'
-		);
-		const coord1 = await paragraph1.clickablePoint();
-		const coord2 = await paragraph2.clickablePoint();
-
-		await page.mouse.move( coord1.x, coord1.y );
-		await page.mouse.down();
-		await page.mouse.move( coord2.x, coord2.y, { steps: 10 } );
-		await page.mouse.up();
-
-		expect( await getSelectedFlatIndices() ).toEqual( [ 1, 2 ] );
-	} );
-
-	it( 'should cut and paste', async () => {
-		await clickBlockAppender();
-		await page.keyboard.type( '1' );
-		await page.keyboard.press( 'Enter' );
-		await page.keyboard.type( '2' );
-		await pressKeyWithModifier( 'primary', 'a' );
-		await pressKeyWithModifier( 'primary', 'a' );
-		await pressKeyWithModifier( 'primary', 'x' );
-
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-
-		await pressKeyWithModifier( 'primary', 'v' );
-
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-	} );
-
-	it( 'should copy and paste', async () => {
-		await clickBlockAppender();
-		await page.keyboard.type( '1' );
-		await page.keyboard.press( 'Enter' );
-		await page.keyboard.type( '2' );
-		await pressKeyWithModifier( 'primary', 'a' );
-		await pressKeyWithModifier( 'primary', 'a' );
-		await pressKeyWithModifier( 'primary', 'c' );
-
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-
-		await page.keyboard.press( 'Backspace' );
-
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-
-		await pressKeyWithModifier( 'primary', 'v' );
-
-		expect( await getEditedPostContent() ).toMatchSnapshot();
-	} );
-
-	it( 'should return original focus after failed multi selection attempt', async () => {
-		await clickBlockAppender();
-		await page.keyboard.type( '1' );
-		await page.keyboard.type( '2' );
-		await page.keyboard.press( 'ArrowLeft' );
```

Comment:
Unfortunately the new test is not doing the same thing. Restored it in #53418. Why not simply copy over the old test as is?